spring-integration-http/src/main/java/org/springframework/integration/http/inbound/HttpRequestHandlingEndpointSupport.java

```diff
@@ -148,8 +148,6 @@
 
 	private volatile Map<String, Expression> headerExpressions;
 
-	private volatile boolean shuttingDown;
```

Comment:
This is still needed - we need to stop new incoming traffic while we are in orderly shutdown mode. i.e. before `stop()` is called.

The MBean exporter calls `beforeShutdown`, waits for in-flight messages to quiesce and then calls `afterShutdown`.

We really need to implement `OrderlyShutdownCapable` on the WS inbound endpoints too.